### DISCUSSION TITLE
LCX Creation Fails when using advanced

### DISCUSSION THREAD

**Discussion by CameronMacG** [tteck/Proxmox#3176] [Help Needed] [answered]
Hi everyone,

First of all, thank you so much for putting these scripts together, they are genuinely brilliant!

I've had an issue with creating LXCs using the scripts for a little while now. I normally create them using the advanced options so I can set a couple of this pre-build, however when I do this the Network test fails after 10 tries and it cancels the install. However, when I use the default settings, the build works with no issues.

This is the output that I got when trying to create the Zabbix LXC;

_____         __    __    _
/__  /  ____ _/ /_  / /_  (_)  __
  / /  / __ `/ __ \/ __ \/ / |/_/
 / /__/ /_/ / /_/ / /_/ / />  <
/____/\__,_/_.___/_.___/_/_/|_|

Using Advanced Settings
Using Distribution: debian
Using debian Version: 12
Using Container Type: 1
Using Root Password: **********
Container ID: 111
Using Hostname: zabbix
Using Disk Size: 25
Allocated Cores: 2
Allocated RAM: 4096
Using Bridge: vmbr1
Using IP Address: dhcp
Using Gateway IP Address: Default
Using APT-Cacher IP Address: Default
Disable IPv6: yes
Using Interface MTU Size: Default
Using DNS Search Domain: Host
Using DNS Server IP Address: Host
Using Vlan: Default
Enable Root SSH Access: yes
Enable Verbose Mode: yes
Creating a Zabbix LXC using the above advanced settings
 ✓ Using PROX for Template Storage.
 ✓ Using local-lvm for Container Storage.
 ✓ Updated LXC Template List
 ✓ LXC Container 111 was successfully created.
 ✓ Started LXC Container
net.ipv6.conf.all.disable_ipv6 = 1
 /✗ No Network! ntainer OS   
✗ No Network After 10 Tries
 🖧  Check Network Settings

Any help would be appreciated!

**Comment by tteck** [comment]
Looks like your DHCP server is not assigning an IP address.

**Reply by CameronMacG** [reply]
DHCP is handled at my router level and is handing out across the rest of my network, I'll take a look at the PVE side and see if i've missed something!

**Reply by CameronMacG** [reply, marked as answer]
Okay, so ultra embarrased at this.

Had a look at the router settings and somehow, IPV6 was enabled. Disabled this and all is happy with my little world again!
